rclcpp/test/executors/test_multi_threaded_executor.cpp

```diff
@@ -34,6 +34,10 @@ class TestMultiThreadedExecutor : public ::testing::Test
   }
 };
 
+constexpr double PERIOD = 1.0f;
+constexpr double TOLERANCE = 0.25f;
+constexpr std::chrono::milliseconds PERIOD_MS = 1000ms;
```

Comment:
Instead of listing the PERIOD twice, can't we do something like:

```
constexpr std::chrono::milliseconds PERIOD_MS = 1000ms;
constexpr double PERIOD = std::chrono::duration_cast<std::chrono::seconds>(PERIOD_MS).count();
constexpr double TOLERANCE = 0.25f;
```

Reply:
See https://github.com/ros2/rclcpp/pull/1105/commits/b013cea1d5778c3688ff5a082bc2866699babc50.

I used:

```cpp
constexpr double PERIOD = PERIOD_MS.count() / 1000.0;
```
as the proposed cast above wouldn't work well if `PERIOD_MS = 100ms`.